modules/registry-refactoring/src/main/resources/META-INF/persistence.xml

```diff
@@ -122,6 +122,9 @@
         <class>org.apache.airavata.registry.core.entities.appcatalog.StorageInterfacePK</class>
```

Comment:
We can remove these Id classes, right?

Reply:
I've removed it. It's a part of the [https://github.com/apache/airavata/pull/167] PR.